README.md

```diff
@@ -16,3 +16,7 @@ To generate the deployment data for a new network the following steps are necess
 - Set `MNEMONIC` in the `.env` file
 - Run `yarn compile <chain_id> [--gasPrice <overwrite_gas_price>] [--gasLimit <overwrite_gas_limit>]`
 
+# For zkSync
+- Set `RPC` in the `.env` file for the new network
```

Comment:
I would only add what is required additionally in the env vars

Reply:
Fixed in below commit.